docs/loading-and-dumping/specific-types-behavior.rst

```diff
@@ -63,7 +63,7 @@ Loader accepts only ``None``, dumper produces no conversion.
 
 bytes-like
 '''''''''''''''''''''''''''''''''''''
-Exact list: ``bytes``, ``bytearray``, ``ByteString``.
+Exact list: ``bytes``, ``bytearray``, ``ByteString``, ``io.BytesIO``, ``typing.IO[bytes]``.
```

Comment:
IO object is not bytes-like. I can not come up with name grouping all of these types so I suggest to add another section ``BytesIO and IO[bytes]``